dashboard/app/models/sections/section.rb

```diff
@@ -193,7 +193,16 @@ def summarize
       linkToAssigned: link_to_assigned,
       numberOfStudents: students.length,
       linkToStudents: "#{base_url}#{id}/manage",
-      sectionCode: code
+      code: code,
+      stage_extras: stage_extras,
+      pairing_allowed: pairing_allowed,
+      login_type: login_type,
+      course_id: course_id,
+      script: {
+        id: script_id,
+        name: script.try(:name),
+      },
```

Comment:
Is there a good reason to return script as an object here, instead of script_id and (if needed) script_name?